admin/acceptance.py

```diff
@@ -456,6 +469,16 @@ def main(reactor, args, base_path, top_level):
 
     runner = options.runner
 
+    from flocker.common.script import eliot_logging_service
+    log_file = open("%s.log" % base_path.basename(), "a")
```

Comment:
I'd rather have the name hard-coded rather than unexpectedly changing when someone renames the file, given that Buildbot is hardcodng it.